src/main/java/io/github/battlepass/quests/service/executor/QuestExecutionBuilderImpl.java

```diff
@@ -0,0 +1,120 @@
+package io.github.battlepass.quests.service.executor;
+
+import io.github.battlepass.objects.quests.variable.ExecutableQuestResult;
+import io.github.battlepass.objects.quests.variable.QuestResult;
+import io.github.battlepass.quests.workers.pipeline.QuestPipeline;
+import io.github.battlepass.service.CheckHelper;
+import org.bukkit.block.Block;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+
+import java.math.BigInteger;
+import java.util.function.UnaryOperator;
+
+public class QuestExecutionBuilderImpl implements QuestExecutionBuilder {
+    private final QuestPipeline pipeline;
+    private Player player;
+
+    private final String questType;
+    private BigInteger progress;
+    private boolean overrideUpdate;
```

Comment:
Conventional practice to sort variables according to their scopes, so

private final
[...]
private final

private
[...]
private